h2/src/main/org/h2/value/ValueLob.java

```diff
@@ -35,6 +35,8 @@
  */
 public class ValueLob extends Value {
 
+    private static final int BLOCK_COMPARISON_SIZE = 512;
```

Comment:
for efficiency, this should probably use the max block size from StreamStore, which is currently 256K

Reply:
ok